samples/ResourceOwnerPasswordFlow/Certificate.cer

```diff
@@ -0,0 +1,24 @@
+Bag Attributes
```

Comment:
You're feedback is priceless. As a lone developer on a small island, I'm not privy to knowing what the future holds. Thank you. 

I'm certainly ready to include ugly hacks in the PR and in the tutorial, to the extent that they're important, which is sounds like they are.